slick/src/main/scala/slick/jdbc/H2Profile.scala

```diff
@@ -123,6 +127,32 @@ trait H2Profile extends JdbcProfile {
       override def valueToSQLLiteral(value: UUID) = "'" + value + "'"
       override def hasLiteralForm = true
     }
+    override val instantType : InstantJdbcType = new InstantJdbcType {
+      val formatter = new DateTimeFormatterBuilder()
+                      .append(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
+                      .optionalStart()
+                      .appendFraction(ChronoField.NANO_OF_SECOND, 0, 9, true)
+                      .optionalEnd()
+                      .appendOffset("+HH", "")
+                      .toFormatter()
+      override def sqlTypeName(sym: Option[FieldSymbol]) = "TIMESTAMP(9) WITH TIME ZONE"
+
+      override def setValue(v: Instant, p: PreparedStatement, idx: Int) : Unit = {
+        p.setString(idx, if (v == null) null else v.toString)
+      }
+      override def getValue(r: ResultSet, idx: Int) : Instant = {
```

Comment:
this is different from the DerbyProfile, which uses `Instant.parse` Why are they different?